examples/hooks/components/HierarchicalMenu.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import {
+  useHierarchicalMenu,
+  UseHierarchicalMenuProps,
+} from 'react-instantsearch-hooks';
+
+import { cx } from '../cx';
+
+export type HierarchicalMenuProps = React.ComponentProps<'div'> &
+  UseHierarchicalMenuProps;
+
+function HierarchicalList({
+  items,
+  refine,
+  createURL,
+}: Pick<
+  ReturnType<typeof useHierarchicalMenu>,
+  'items' | 'refine' | 'createURL'
+>) {
+  return (
+    <ul className="ais-HierarchicalMenu-list">
+      {items.map((item) => (
+        <li
+          key={item.value}
+          className={cx(
+            'ais-HierarchicalMenu-item',
+            item.isRefined && 'ais-HierarchicalMenu-item--selected'
+          )}
+        >
+          <a
+            className="ais-HierarchicalMenu-link"
+            href={createURL(item.value)}
+            onClick={(e) => {
+              e.preventDefault();
```

Comment:
For explicitness and consistency?

```suggestion
            onClick={(event) => {
              event.preventDefault();
```